installer/numa-topo.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: resource-exporter
+  namespace: kube-system
+rules:
+  - apiGroups: ["apiextensions.k8s.io"]
+    resources: ["customresourcedefinitions"]
+    verbs: ["create", "get", "list", "watch", "delete"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["list", "watch"]
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["list", "watch"]
+  - apiGroups: ["nodeinfo.volcano.sh"]
+    resources: ["numatopologies"]
+    verbs: ["create", "delete", "get", "list", "patch", "update", "watch"]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: resource-exporter-bind
+subjects:
+  - kind: ServiceAccount
+    name: resource-exporter-account
+    namespace: kube-system
+roleRef:
+  kind: ClusterRole
+  name: resource-exporter
+  apiGroup: rbac.authorization.k8s.io
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: resource-exporter-account
+  namespace: kube-system
+
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: resource-exporter-daemonset
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      name: resource-topology
+  template:
+    metadata:
+      labels:
+        name: resource-topology
+    spec:
+      serviceAccountName: resource-exporter-account
+      containers:
+        - name: resource-topology
+          image: volcanosh/numatopo:latest
+          imagePullPolicy: IfNotPresent
+          args:
+            - --logtostderr
+            - --kubelet-conf=/host/kubeletconf/config.yaml

```

Comment:
"/host/kubeletconf/config.yaml " is not a valid path, suggest change it to "/var/lib/kubelet/config.yaml"